panoptes_aggregation/tests/batch_aggregation/test_batch_aggregation.py

```diff
@@ -0,0 +1,124 @@
+import unittest
+from unittest.mock import patch, MagicMock, call
+from panoptes_aggregation.scripts import batch_utils
+from panoptes_aggregation.batch_aggregation import run_aggregation
+from panoptes_aggregation import batch_aggregation as batch_agg
+
+wf_export = 'panoptes_aggregation/tests/batch_aggregation/wf_export.csv'
+cls_export = 'panoptes_aggregation/tests/batch_aggregation/cls_export.csv'
+
+
+@patch("panoptes_aggregation.batch_aggregation.BatchAggregator._connect_api_client", new=MagicMock())
+class TestBatchAggregation(unittest.TestCase):
+    @patch("panoptes_aggregation.batch_aggregation.workflow_extractor_config")
+    @patch("panoptes_aggregation.batch_aggregation.BatchAggregator")
+    def test_run_aggregation(self, mock_aggregator, mock_wf_ext_conf):
+        mock_aggregator.process_wf_export.return_value = MagicMock()
+        mock_aggregator.process_cls_export.return_value = MagicMock()
```

Comment:
I am 80% sure that any method on a `MagicMock()` returns a `MagicMock()` by default, so these lines might not be needed.

Reply:
True!